Add changelog (#677)
equinor/semeio: CHANGELOG.md

# CHANGELOG

## [1.18.0]

- Moved `fmudesign` from `fmu-tools` to `semeio` and made major improvements to the sampling engine as documentet below.

### Major Improvements in fmudesign's Sampling Engine

> The next version of `fmudesign` introduces major improvements to its sampling engine:

Reviewer: ```suggestion
This version of `fmudesign` introduces major improvements to its sampling engine:
```
